internal/clients/team5/voting.go

```diff
@@ -46,7 +47,7 @@ func findIndexOfScore(opinionSortByScore []opinionScore, opinionSortByTeam []opi
 }
 
 // translate int to shared.clientID but put our ID first and someone last
-func sortedMapOfOpinion(rank []int) (sortedMap []shared.ClientID) {
+func (c *client) sortedMapOfOpinion(rank []int) (sortedMap []shared.ClientID) {
 	sortedMap = append(sortedMap, shared.Team5)
 	for i := 0; i < len(rank); i++ {
 		if rank[i] == 0 {
```

Comment:
Not really a fan of this hard-coded if-else approach but guess we can leave for now